project/bhchen2001/README.rst

```diff
@@ -0,0 +1,95 @@
+========================
+Parallel Vonoroi Diagram
+========================
+
+Basic Information
+=================
+
+Github Repository: https://github.com/bhchen2001/Parallel_Vonoroi_Diagram
+
+Vonoroi diagram is a partitioning of a plane into regions based on distance to a specific points,
+which is widely used in many fields, such as computer graphics and face recognition.
+There are some classic algorithms to generate vonoroi diagram, such as divide and conquer method, fortune's algorithm,
+and their time complexity is O(nlogn).
+However, with large-scale data, these algorithms will become time-consuming.
+
+Problem to Solve
+=================
+
+The primary goal if this project is to implement a thread-level parallel version of 
+two-dimensional vonoroi diagram algorithm on shared-memory multi-core machines.
+
+For the parallel version, the following requirements should be satisfied:
+
+1. A high-speed data structure for storing the vonoroi diagram and searching the nearby points.
+2. K-d tree for decomposing the points into different blocks.
```

Comment:
Did you have experience with a spatial index?  Are you familiar with any existing source code implementing a k-d tree?

A k-d tree alone suffices a term project.